Converter/Calculator/Converter.cs

```diff
@@ -0,0 +1,140 @@
+
+using System.Diagnostics;
+
+namespace Calculator;
+
+public static class Converter
+{
+    public static void ConvertTo(int value)
+    {
+        while(true)
+        {
+            int number, b;
+            bool i;
+            do
+            {
+                Console.WriteLine("\n Choose a number system:");
+                Console.WriteLine(" For system 2 write 2");
+                Console.WriteLine(" For system 16 write 16");
+                Console.WriteLine(" For system 8 write 8");
+                Console.WriteLine(" To enter a new number, type 6");
+                Console.WriteLine("\n Press 0 to exit");
+                i = int.TryParse(Console.ReadLine(), out number);
+            } while (i == false);
+
+
+            switch (number)
+            {
+                case 2:
+                    ReverseStr(ConvertToBin(value));
+                    break;
+                case 16:
+                    ReverseStr(Converter.ConvertToHex(value));
+                    break;
+                case 8:
+                    ReverseStr(Converter.ConvertToOct(value));
+                    break;
+                case 6:
+                    Console.Write("\n Write a number to convert: ");
+                    var c = int.TryParse(Console.ReadLine(), out b);
+                    break;
+                case 0:
+                    return;
+                default:
+                    Console.WriteLine("Try Again, incorrect value");
+                    break;
+            }
+        }
+    }
+
+    private static void ReverseStr(string str)
```

Comment:
This method should return something because its responsibility is to reverse strings

Reply:
Or you could give it name ReverseAndPrint (but this is bad option)